src/einsteinpy/symbolic/ricci.py

```diff
@@ -96,3 +154,39 @@ def symbols(self):
 
         """
         return self.syms
+
+
+class RicciScalar:
+    """
+    Class for defining Ricci Scalar
+    """
+
+    def __init__(self, expression, syms, parent_metric=None):
+        """
+        Constructor and Initializer
+
+        Parameters
+        ----------
+        expression : ~sympy expression
+            Simple sympy
+        syms : tuple or list
+            Tuple of crucial symbols dentoting time-axis, 1st, 2nd, and 3rd axis (t,x1,x2,x3)
+        parent_metric : ~einsteinpy.symbolic.metric.MetricTensor or None
+            Corrosponding Metric for the Ricci Tensor.
+            None if it should inherit the Parent Metric of Christoffel Symbols.
+            Defaults to None.
```

Comment:
```
Corrosponding Metric for the Ricci Scalar.
Defaults to None.
```
Remove the middle line

Reply:
The main classmethod would be `from_riccitensor(cls, riccitensor, parent_metric=None)` . 

`from_metric` can call `from_christoffel`, `from_christoffel` can call `from_riemann` and `from_riemann` can call `from_riccitensor`.

FYI, ricci scalar is tensor contraction of Ricci tensor with config `ul`.

Reply:
Yes, the mathematical development is clear to me. Anyway, I upload these docstring modifications before uploading the classmethod.